Index: b/tests/Makefile.am
===================================================================
--- a/tests/Makefile.am	2012-04-16 03:11:53.000000000 +0000
+++ b/tests/Makefile.am	2013-12-15 14:55:29.795240522 +0000
@@ -19,7 +19,7 @@
 
 tests_validate_SOURCES = tests/validate.c tests/validate.h
 tests_validate_CPPFLAGS = $(AM_CPPFLAGS)
-tests_validate_LDADD = $(MAGICKCORE_LIBS) $(MAGICKWAND_LIBS)
+tests_validate_LDADD = $(MAGICKCORE_LIBS) $(MAGICKWAND_LIBS) $(MATH_LIBS)
 
 TESTS_XFAIL_TESTS = 
 
Index: b/wand/Makefile.am
===================================================================
--- a/wand/Makefile.am	2012-02-19 19:56:56.000000000 +0000
+++ b/wand/Makefile.am	2013-12-15 14:57:10.625250605 +0000
@@ -139,11 +139,11 @@
 
 wand_drawtest_SOURCES      = wand/drawtest.c
 wand_drawtest_LDFLAGS      = $(LDFLAGS)
-wand_drawtest_LDADD        = $(MAGICKWAND_LIBS)
+wand_drawtest_LDADD        = $(MAGICKCORE_LIBS) $(MAGICKWAND_LIBS)
 
 wand_wandtest_SOURCES      = wand/wandtest.c
 wand_wandtest_LDFLAGS      = $(LDFLAGS)
-wand_wandtest_LDADD        = $(MAGICKWAND_LIBS)
+wand_wandtest_LDADD        = $(MAGICKCORE_LIBS) $(MAGICKWAND_LIBS)
 
 WAND_CHECK_PGRMS = \
 	wand/drawtest \
Index: b/Magick++/Makefile.am
===================================================================
--- a/Magick++/Makefile.am	2011-10-28 18:34:50.000000000 +0000
+++ b/Magick++/Makefile.am	2013-12-15 15:05:33.605289749 +0000
@@ -188,67 +188,67 @@
 	Magick++/tests/readWriteImages
 
 Magick___demo_analyze_SOURCES		= Magick++/demo/analyze.cpp
-Magick___demo_analyze_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_analyze_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_button_SOURCES		= Magick++/demo/button.cpp
-Magick___demo_button_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_button_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_demo_SOURCES		= Magick++/demo/demo.cpp
-Magick___demo_demo_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_demo_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_detrans_SOURCES		= Magick++/demo/detrans.cpp
-Magick___demo_detrans_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_detrans_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_flip_SOURCES		= Magick++/demo/flip.cpp
-Magick___demo_flip_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_flip_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_gravity_SOURCES		= Magick++/demo/gravity.cpp
-Magick___demo_gravity_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_gravity_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_piddle_SOURCES		= Magick++/demo/piddle.cpp
-Magick___demo_piddle_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_piddle_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_shapes_SOURCES		= Magick++/demo/shapes.cpp
-Magick___demo_shapes_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_shapes_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___demo_zoom_SOURCES		= Magick++/demo/zoom.cpp
-Magick___demo_zoom_LDADD		= $(MAGICKPP_LIBS)
+Magick___demo_zoom_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_appendImages_SOURCES	= Magick++/tests/appendImages.cpp
-Magick___tests_appendImages_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_appendImages_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_attributes_SOURCES	= Magick++/tests/attributes.cpp
-Magick___tests_attributes_LDADD		= $(MAGICKPP_LIBS)
+Magick___tests_attributes_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_averageImages_SOURCES	= Magick++/tests/averageImages.cpp
-Magick___tests_averageImages_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_averageImages_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_coalesceImages_SOURCES	= Magick++/tests/coalesceImages.cpp
-Magick___tests_coalesceImages_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_coalesceImages_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_coderInfo_SOURCES	= Magick++/tests/coderInfo.cpp
-Magick___tests_coderInfo_LDADD		= $(MAGICKPP_LIBS)
+Magick___tests_coderInfo_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_color_SOURCES		= Magick++/tests/color.cpp
-Magick___tests_color_LDADD		= $(MAGICKPP_LIBS)
+Magick___tests_color_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_colorHistogram_SOURCES	= Magick++/tests/colorHistogram.cpp
-Magick___tests_colorHistogram_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_colorHistogram_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_exceptions_SOURCES	= Magick++/tests/exceptions.cpp
-Magick___tests_exceptions_LDADD		= $(MAGICKPP_LIBS)
+Magick___tests_exceptions_LDADD		= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_montageImages_SOURCES	= Magick++/tests/montageImages.cpp
-Magick___tests_montageImages_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_montageImages_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_morphImages_SOURCES	= Magick++/tests/morphImages.cpp
-Magick___tests_morphImages_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_morphImages_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_readWriteBlob_SOURCES	= Magick++/tests/readWriteBlob.cpp
-Magick___tests_readWriteBlob_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_readWriteBlob_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 Magick___tests_readWriteImages_SOURCES	= Magick++/tests/readWriteImages.cpp
-Magick___tests_readWriteImages_LDADD	= $(MAGICKPP_LIBS)
+Magick___tests_readWriteImages_LDADD	= $(MAGICKPP_LIBS) $(MAGICKCORE_LIBS)
 
 
 MAGICKPP_LOCAL_TARGETS = www/Magick++/NEWS.html www/Magick++/ChangeLog.html
